test/metabase/db/liquibase_test.clj

```diff
@@ -53,14 +53,17 @@
   "Read a liquibase migration file and returns all the migration id that is applied to `db-type`.
   Ids are orderer in the order it's defined in migration file."
   [file-path db-type]
-  (let [content (u.yaml/from-file (io/resource file-path))]
+  (let [content (u.yaml/from-file (io/resource file-path))
```

Comment:
Since this no longer applies, we should be able to just delete the changeset from 000_legacy_migrations.yaml rather than add the programatic removal?

Reply:
Done